index.html

```diff
@@ -214,7 +214,25 @@ <h3>Skills</h3>
          </ul>
       </div>
     </section>
-    <section class="contact-me">
-    </section>
+  <section id="contact-me">
+    <div class="contact-me-background">
+    <h2 class="header-contact-me">
+      Contact Me 
+    </h2> 
+    <p class="contact-me-paragraph">
+      If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+    </p>
+    <form id="contact-me-form" action="https://formspree.io/f/xbjeygan" method="post">
+      <input class="contact-me-name" type="text" name="name" placeholder="Hernán" required maxlength="30" minlength="5"/>
+      <input class="contact-me-email" type="email" name="email" placeholder="hernanguemes@hotmail.com" name="contact-email-address" required>
+      </label>
+      <textarea class="contact-me-textarea" placeholder="Write your message here" required cols="30" rows="8" maxlength="500" minlength="5"></textarea>
+      <div class="action-button">
+        <button class="contact-me-submit-button" type="submit">Get in touch</button>
+      </div> 
+      <span class="closing-line"><img src="Images/Shape.png"></span>   
+    </form>  
+    </div>
```

Comment:
Well done on this beautiful project.  Just a few things will make this project even better 👇🏾 

- [ ] Looking at the image below, the input element currently has a border as highlighted in the image. Please ensure that this border matches the Figma design. It appears that this input element should not have a border or outline at all. Kindly fix this.

![image](https://user-images.githubusercontent.com/84156714/222014730-bc440982-31be-48b2-991e-c539db02fb65.png)

Reply:
Hi there,
Thanks for the feedback! I'm resending the code for your review with the changes requested applied.
-set outline inside the form to none

Thank you in advance!